examples/get_target_descriptions.py

```diff
@@ -45,6 +46,7 @@ def main():
     #         "track_duration":20.0
     #     },
     #     {
+    # input type:
```

Comment:
Where does this line come from?  It doesn't look like part of the example output.